.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1 @@
+**Description**
```

Comment:
This has to have few cafeterias which is the purpose of having a template. You may get some ideas from this https://github.com/99xt/serverless-dynamodb-local/blob/v1/.github/PULL_REQUEST_TEMPLATE